src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -397,6 +399,10 @@ def create_or_update_test_with_config(
             new_body["keyvaultReferenceIdentityType"] = IdentityType.SystemAssigned
             new_body.pop("keyvaultReferenceIdentityId")
     subnet_id = subnet_id or yaml_test_body.get("subnetId")
+    if disable_public_ip:
+        new_body["publicIPDisabled"] = disable_public_ip
+    else:
+        new_body["publicIPDisabled"] = yaml_test_body.get("publicIPDisabled", False)
```

Comment:
I think this should be `None`. Afaik, sending it for public load test will throw an error (even if we send `False`). Please verify

Reply:
No Mitesh, we can set the value to false and default behavior for portal is also same, the request has publicIPDisabled as false